.projenrc.js

```diff
@@ -0,0 +1,101 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
+const { AwsCdkConstructLibrary, ProjectType, NodePackageManager } = require("projen");
+
+const project = new AwsCdkConstructLibrary({
+  name: "datadog-cdk-constructs",
+  description: "CDK Construct library to automatically instrument python and node functions with datadog tracing",
```

Comment:
- Nit: capitalize Library, Python, Node, and Datadog
- Instead of "functions" I would say "Lambda functions"
- Our instrumentation can help collect metrics and logs as well. So I would just say "instrument... with Datadog" rather than mention "Datadog tracing" specifically